benchmarks/README.md

```diff
@@ -0,0 +1,56 @@
+# PermaplanT Performance Audit
+
+## Requirements
+
+-   nodejs 19.4.0
+-   npm
+
+## Installation and Usage
+
+1. Install dependencies
+
+```shell
+npm install
+```
+
+2. Start the backend from the backend folder
+
+```shell
+cargo run
+```
+
+3. Start the frontend from the frontend folder
+
+```shell
+npm run dev
+```
+
+4. Run benchmarks
+
+```shell
+npm run benchmark
+```
+
+## Benchmarking
+
+The benchmarking script runs a performance audit on a web page using `Lighthouse` and `Playwright`.
+The audit measures the performance of a web page by generating a performance score and then saves the results of the audit in `report` folder:
+
+-   `<test_name>-report.json` - the raw report of LightHouse for a single test case defined in `performance-audit.spec.js`
+-   `<timestamp>-lighthouse-results.csv` - the results of the audit for all test cases
```

Comment:
Why not have test_name here, too?

Reply:
the mentioned file is the result of an entire test run i.e. all test cases and not a single specific test case